src/Libraries/RevitNodesUI/RevitDropDown.cs

```diff
@@ -62,6 +62,17 @@ public override void Dispose()
             DynamoRevitApp.EventHandlerProxy.DocumentOpened -= Controller_RevitDocumentChanged;
             base.Dispose();
         }
+
+        public Boolean CanBuildOutputAst(string itemValue = null, string selectedValue = null)
```

Comment:
Rename itemValue => itemValueToIgnore, selectedValue => selectedValueToIgnore, and add comments for this method.